srcpkgs/black/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'black'
 pkgname=black
-version=19.10b0
-revision=3
+version=20.8b1
+revision=2
 build_style=python3-module
-pycompile_module="black.py blib2to3 _black_version.py"
 hostmakedepends="git python3-setuptools"
```

Comment:
Drop `git` dependency and replace `python3-setuptools` with `python3-setuptools_scm` because this package is required during build for version detection.